docs/src/modules/java/pages/timers.adoc

```diff
@@ -0,0 +1,152 @@
+= Timers
+:page-supergroup-java-scala: Language
+
+Timers allow for scheduling calls in the future. For example, to verify that some process have been completed or not.
+
+Timers are persisted in the https://docs.kalix.io/reference/glossary.html#proxy[Kalix proxy] and are guaranteed to run at least once. 
+
+When a timer is triggered, the scheduled call is executed. If successfully excuted, the timer completes and is automatically removed. In case of a failure, the timer is rescheduled to run once more after 3 seconds. This process repeats until the call succeeds.
+
+You can schedule a timer for any service method in Kalix, but you can only create a timer from within an Action and by passing a `DefferedCall`. 
+
+**Timer features**:
+
+* Timers are guaranteed to run at least once.
+* Timers can be scheduled to run at any time in the future.
+* Timers can be cancelled.
+* Timers are automatically removed once successfully completed.
+* Timers are re-scheduled in case of failures.
+
+**Timer limitations**: 
+
+* Timers run from the Kalix proxy and therefore won't be run if the service is scaled down to zero instances.
```

Comment:
Do we talk about "Kalix proxy" in the documentation? I don't think that is something we should describe too much for the user. 

Currently, it's not possible to scale to zero. Later that might be possible, but do we need to document that now?

Reply:
We mention the proxy a few times in the docs and we even have a glossary entry for it. 
I can remove the mention, no problem.